db/schema.rb

```diff
@@ -43,10 +43,11 @@
   add_index "available_points", ["exercise_id", "name"], name: "index_available_points_on_exercise_id_and_name", unique: true, using: :btree
 
   create_table "awarded_points", force: :cascade do |t|
-    t.integer "course_id",     null: false
-    t.integer "user_id",       null: false
-    t.integer "submission_id"
-    t.string  "name",          null: false
+    t.integer  "course_id",     null: false
+    t.integer  "user_id",       null: false
+    t.integer  "submission_id"
+    t.string   "name",          null: false
+    t.datetime "created_at"
```

Comment:
Here there was only a comma if there was further details about the element. Should I still place a comma?